### PR TITLE
 FIX: fix display bugs in visual tasks

### LINKED ISSUE #9: Add a reminder to leave the alarm button on their tummy in the beginning of the PCT

### LINKED ISSUE #12: Fixation point on the visual stimuly (grating pattern) of the PCT is not round
Please fix proportions

### LINKED ISSUE #14: Fixation point of the ET calibration is tiny (at least for the PCT, but may happen to all tasks)

### LINKED ISSUE #15: ET calibration should be run for all tasks
We could run a nine or thirteen points before the PCT and three or five points in resting and BHT (to speed up once it was calibrated once).

### LINKED ISSUE #18: Add fixation point BEFORE and AFTER the resting state
In order to re-calibrate/estimate drift of the ET

### PR DESCRIPTION
**Pull request by esavary**
- Add fixation points before and after movie in RS task in order to estimate drift of the ET. closes #18 
- Fix the size of the fixation points in calibration + set 9 points calibration for PCT and 5 points calibration for RS and BHT. closes #15 , closes #14 
- Improve text at the beginning in PCT + add reminder to left the alarm button on the tummy. closes #9
- Fix the roundness of the fixation point on top of the grating in PCT. closes #12

### CODE REVIEW

File: task-bht_bold.psyexp [hunk 1]
```diff
@@ -1031,16 +1031,16 @@
...
       <Param name="movementDur" updates="None" val="1.0" valType="num"/>
       <Param name="name" updates="None" val="calibration_2" valType="code"/>
-      <Param name="outerRadius" updates="None" val="0.015" valType="num"/>
+      <Param name="outerRadius" updates="None" val="0.75" valType="num"/>
       <Param name="progressMode" updates="None" val="time" valType="str"/>
       <Param name="randomisePos" updates="None" val="True" valType="bool"/>
       <Param name="targetDelay" updates="None" val="1.0" valType="num"/>
       <Param name="targetDur" updates="None" val="1.5" valType="num"/>
-      <Param name="targetLayout" updates="None" val="NINE_POINTS" valType="str"/>
+      <Param name="targetLayout" updates="None" val="FIVE_POINTS" valType="str"/>
```
Comment (oesteban) [inline task-bht_bold.psyexp:1043]: why this change?

File: task-bht_bold.psyexp [hunk 2]
```diff
@@ -1031,16 +1031,16 @@
       <Param name="innerBorderColor" updates="None" val="black" valType="color"/>
       <Param name="innerBorderWidth" updates="None" val="2" valType="num"/>
       <Param name="innerFillColor" updates="None" val="green" valType="color"/>
-      <Param name="innerRadius" updates="None" val="0.0075" valType="num"/>
+      <Param name="innerRadius" updates="None" val="0.35" valType="num"/>
```
Comment (oesteban) [inline task-bht_bold.psyexp:1034]: I would need to see this on a 800x600 screen. Looks like the dot is going to be huge now (which is worse than too small)

File: task-bht_bold.psyexp [hunk 3]
```diff
@@ -1031,16 +1031,16 @@
...
       <Param name="movementAnimation" updates="None" val="True" valType="bool"/>
       <Param name="movementDur" updates="None" val="1.0" valType="num"/>
       <Param name="name" updates="None" val="calibration_2" valType="code"/>
-      <Param name="outerRadius" updates="None" val="0.015" valType="num"/>
+      <Param name="outerRadius" updates="None" val="0.75" valType="num"/>
```
Comment (oesteban) [inline task-bht_bold.psyexp:1038]: Ditto

File: task-pct_bold.psyexp [hunk 4]
```diff
@@ -632,19 +632,19 @@
...
+        <Param val="(0.0, 0.6)" valType="list" updates="constant" name="pos"/>
         <Param val="True" valType="bool" updates="None" name="saveStartStop"/>
         <Param val="" valType="code" updates="None" name="startEstim"/>
         <Param val="time (s)" valType="str" updates="None" name="startType"/>
         <Param val="0.0" valType="code" updates="None" name="startVal"/>
         <Param val="duration (s)" valType="str" updates="None" name="stopType"/>
         <Param val="" valType="code" updates="constant" name="stopVal"/>
         <Param val="True" valType="bool" updates="None" name="syncScreenRefresh"/>
-        <Param val="In this task you will see a grating patterns on the screen. Please keep your eyes on the red fixation circle like the one below." valType="str" updates="constant" name="text"/>
+        <Param val="In this task you will do several activities. Whenever you see a green circle like the one below, please fix your gaze on it. If the circle moves, follow it with your eyes." valType="str" updates="constant" name="text"/>
```
Comment (oesteban) [inline task-pct_bold.psyexp:647]:
```suggestion
        <Param val="This task is composed of several blocks of stimuli. Whenever you see a green circle like the one below, please fix your gaze on it. If the circle moves, follow it with your eyes." valType="str" updates="constant" name="text"/>
```

File: task-pct_bold.psyexp [hunk 5]
```diff
@@ -696,7 +696,7 @@
         <Param val="duration (s)" valType="str" updates="None" name="stopType"/>
         <Param val="" valType="code" updates="constant" name="stopVal"/>
         <Param val="True" valType="bool" updates="None" name="syncScreenRefresh"/>
-        <Param val="When the red fixation point moves, please follow it with your eyes." valType="str" updates="constant" name="text"/>
+        <Param val=" If you see &quot;RIGHT&quot; or &quot;LEFT&quot;, tap your thumb and the other fingers of the corresponding hand.&amp;#10;Please leave the alarm button on your tummy." valType="str" updates="constant" name="text"/>
```
Comment (oesteban) [inline task-pct_bold.psyexp:699]:
```suggestion
        <Param val=" If you see &quot;RIGHT&quot; or &quot;LEFT&quot;, sequentially tap your thumb with your other fingers of the corresponding hand.&amp;#10;Please leave the alarm button somewhere you can retrieve it when the task is concluded." valType="str" updates="constant" name="text"/>
```

File: task-rest_bold.psyexp [hunk 6]
```diff
@@ -78,7 +78,7 @@
         <Param val="time (s)" valType="str" updates="None" name="startType"/>
         <Param val="0.0" valType="code" updates="None" name="startVal"/>
         <Param val="duration (s)" valType="str" updates="None" name="stopType"/>
-        <Param val="1200" valType="code" updates="constant" name="stopVal"/>
+        <Param val="1206" valType="code" updates="constant" name="stopVal"/>
```
Comment (oesteban) [inline task-rest_bold.psyexp:81]: Am I correctly understanding you are adding 3 seconds for the fixation point at the beginning and end?

File: task-bht_bold.psyexp (hunk 1 above)
Comment (esavary) [reply task-bht_bold.psyexp:1043]: To speed up the calibration for BHT and RS (as suggested in #15 ), but we can also set it to 9 points for the 3 tasks.

**Review by oesteban — APPROVED** [review]

File: task-rest_bold.psyexp (hunk 6 above)
Comment (esavary) [reply task-rest_bold.psyexp:81]: Yes, exactly.
